packages/components/src/components/callout/callout.tsx

```diff
@@ -51,8 +51,6 @@ export class Callout {
       characterData: true,
     });
     this.mo = observer;
-    // Wait for full styles before measuring
-    window.requestAnimationFrame(this.adjustSize);
```

Comment:
Let's also remove the `adjustSize` method and the entire MutationObserver stuff… and also make the value of `aspect-ratio` in CSS `1 / 1` instead of `1` (not sure if that really makes a difference, it's only a hunch)

Reply:
removed